### DISCUSSION TITLE
Mountain Duck UI Questions

### DISCUSSION THREAD

**Discussion by WhatImListnTo** [iterate-ch/cyberduck#16137] [Q&A]
I am confused by what I see from Mountain Duck.  Would anyone please explain what these files at the top of this pop-up are supposed to represent?

![OI6oALYGjI_13_11_02_14](https://github.com/user-attachments/assets/c4985123-773b-4e49-a8de-8a1452e5db36)
 
There are ten files listed above Recent files that all show Unknown below them.  They seem to be stuck there, but I do not know what Mountain Duck is trying to tell me.

**Comment by tatze96** [comment]
The section above the *Recent files* section shows the synchronization progress and pending changes after the current transfer. For more information, refer to our [wiki page](https://docs.cyberduck.io/mountainduck/connect/sync/#sync-progress).

Regarding the "Unknown" below the items:
Mountain Duck will display the writing application there unless it is Mountain Duck itself.
